src/containerapp/azext_containerapp/_help.py

```diff
@@ -1219,120 +1219,120 @@
           az containerapp env java-component list -g MyResourceGroup --environment MyEnvironment
 """
 
-helps['containerapp env java-component spring-cloud-config'] = """
+helps['containerapp env java-component config-server-for-spring'] = """
```

Comment:
`az containerapp env java-component spring-cloud-config` has been released since `0.3.48`, it should be deprecated first.

Reply:
You can deprecate the command as this PR: https://github.com/Azure/azure-cli-extensions/commit/fbb8641c3ce5877d1a77cde506a42ab7dc72991c